packages/kbn-search-api-panels/components/code_box.tsx

```diff
@@ -81,7 +81,8 @@ export const CodeBox: React.FC<CodeBoxProps> = ({
     <EuiThemeProvider colorMode="dark">
       <EuiButtonEmpty
         aria-label={i18n.translate('searchApiPanels.welcomeBanner.codeBox.selectAriaLabel', {
-          defaultMessage: 'Select a programming language',
+          defaultMessage:
+            'Select a programming language from the list of available languages to show a code snippet in that language',
```

Comment:
@rshen91, I apologize for adding a comment to your draft PR. But I believe it's important for us to reach a consensus on the final appearance + I see related to that issues which we also should fix. I understand @dave-gus's point about not adding more text to the aria-label. We should aim to avoid elements with similar aria-labels.

For this case, I suggest using: `Select a programming language for the {context} code snippet.`

Reply:
e.g. for 
<img width="1215" alt="image" src="https://github.com/user-attachments/assets/8f8411a3-29d1-49cd-865e-876d0e77e0b2">

`Select a programming language for the "Configure your client" code snippet.`

@dave-gus please confirm

Reply:
I'm in agreement with @alexwizp. His recommendation provides the missing context @dave-gus called out in the original issue and is still fairly compact.